projects/components/src/overlay/overlay.service.ts

```diff
@@ -13,16 +13,22 @@ import { SheetOverlayComponent } from './sheet/sheet-overlay.component';
   providedIn: 'root'
 })
 export class OverlayService implements OnDestroy {
-  private activePopover?: PopoverRef;
-  private closeSubscription?: Subscription;
+  private activeSheetPopover?: PopoverRef;
```

Comment:
Should these just be separate services at this point? How much is shared?

Reply:
We can make it separate as well. It just delegates to popover. But it doesn't seem wrong for them to be in the same service either.

Reply:
Will prefer to keep it in the same service for now.